dev-tools/kubernetes/filebeat/Dockerfile.debug

```diff
@@ -1,4 +1,4 @@
-FROM golang:alpine3.15 as builder
```

Comment:
Alpine Linux isn't actually in our support matrix (https://www.elastic.co/support/matrix), so using Debian as the base image here makes more sense anyway.